internal/discovery/cluster.go

```diff
@@ -38,9 +38,16 @@ func (c ClusterDiscovery) GetInterval() time.Duration {
 // Execute one iteration of a discovery and publish the results to the collector
 func (c ClusterDiscovery) Discover() (string, error) {
 	cluster, err := cluster.NewCluster()
+
 	if err != nil {
 		log.Debugf("Error creating the cluster data object: %s", err)
-		return "No HA cluster discovered on this host", nil
+	}
+
+	// If no cluster is found, the discovery payload is sent anyway.
```

Comment:
I don't know if we should print this here.
In the next lines we would have a strange `"Cluster with name:  successfully discovered"` in this scenario.
What about moving this to the end of the file like we do for the SAPSystem discovery:
```
if cluster == nil {
    return "No HA cluster discovered on this host", nil
}

return fmt.Sprintf("Cluster with name: %s successfully discovered", cluster.Name), nil
```

Reply:
oh right i;ve missed this